Добавление методов для поддержки GXTextField

Drogonov opened the pull request:
Добавил дополнительно extensions, а также несколько protocols и enums которые мы часто используем в своих проектах
Комментарии пока на английском, как переделаю их в GXTextField переведу их на русский

Drogonov committed:
Добавил для текстфилдов данные

WeDontNeedTo commented on Sources/GXUtilz/Common/String.swift:
```diff
@@ -8,6 +8,44 @@
+        let numbersReplacing = "[^0-9]"
+        var textReplacing = "[^А-Яа-я]"
+        
+        switch formatLanguage {
```
Что, если нужно будет сортировать и русские, и английские?

Drogonov committed:
Пофиксил функции работающие с текстом и поменял описание на русский
Но onlyNumbers нужно переделать на enum как мне кажется я это реализую в текстфилдах и потом сделаю еще один пулл реквест в утилиты

WeDontNeedTo commented:
LGTM!
